lib/reek/ast/object_refs.rb

```diff
@@ -1,33 +1,30 @@
 module Reek
-  module AST
+  module AST  # :nodoc:
```

Comment:
Without this (or `# rubocop:disable Style/Documentation`) RuboCop says `AST` is not documented. This is probably a bug in RuboCop – it is triggered by the newly-added `ObjectRef`, and it seems RuboCop no longer treats `AST` as a namespace-only module.

I’m ok with any other way to stop RuboCop failing on undocumented `AST` that you prefer, I just want it to not be picky about it.